app/views/events/_event.html.erb

```diff
@@ -168,3 +139,29 @@
     <p><%= simple_format(@event.ticket_information) %></p>
   </div>
 </div>
+
+<p>If you have any questions about volunteering, feel free to email <a href="mailto:volunteers@londondecom.org">volunteers@londondecom.org</a></p>
+
+<% if current_user.ticket_type != :general || @event.available_tickets_for_code(current_user.membership_number) > 0 %>
+<script src="https://www.eventbrite.com/static/widgets/eb_widgets.js"></script>
+<script type="text/javascript">
+    var orderCompleteCallback = function() {
```

Comment:
I think this only gets called if they close the modal box? What happens if they buy the ticket and the quit the page? I suppose it will not show them volunteer options for 30 mins which isn't the end of the world?

Reply:
It gets called when the purchase is completed and only then.

It would still be called as the purchase has been completed. There is such a slim chance you'd be actually able to close the window before the sale had completed too.

Just tested with a few accounts and even when I tried to close it still purchased the ticket and did the cache invalidation